README.md

```diff
@@ -71,68 +71,39 @@ This will automatically configure the database tables and their settings for us.
    file as `SUPABASE_URL`, `SUPABASE_ANON_KEY`, and `SUPABASE_SERVICE_KEY`,
    respectively.
 
-### Payments
-
-1. Create the “Premium tier” product in Stripe via the Stripe CLI:
-
-```
-stripe products create \
-  --name="Premium tier" \
-  --default-price-data.unit-amount=500 \
-  --default-price-data.currency=usd \
-  --default-price-data.recurring.interval=month \
-  --description="Unlimited todos"
-```
+### Payments and Subscriptions
 
-2. The resulting [product object](https://stripe.com/docs/api/products) will be
-   printed in the terminal. Copy the `default_price` value as
-   `STRIPE_PREMIUM_PLAN_PRICE_ID` in [`constants.ts`](constants.ts).
+1. Copy your Stripe secret key as `STRIPE_SECRET_KEY` into your `.env` file. We
+   recommend using the test key for your development environment.
+2. Run `deno task init:stripe` and follow the instructions. This automatically
+   creates your "Premium tier" product and configures the Stripe customer
+   portal.
 
-3. Next, head over to
-   [your Stripe dashboard settings](https://dashboard.stripe.com/test/apikeys)
-   to copy the `STRIPE_SECRET_KEY` into your `.env` file. We recommend using the
-   test key for your development environment.
+> Note: go to [init/stripe.ts] if you'd like to learn more about how the
```

Comment:
@niklasmtj, regarding https://github.com/denoland/saaskit/pull/93#discussion_r1168232989, I'd rather have one or the other. I've added this comment to point people in the right direction if they want to learn more. At some point, I'll include technical details as JSDoc comments in the implementation to give an even more precise idea of what's happening. Do you think this addresses your concern?

Reply:
Sounds good to me 👍🏼